src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportIntegrationTest.java

```diff
@@ -46,66 +61,73 @@ public String getTestedClassName() {
     }
 
     @Test
-    public void testGenomicsDBImportFileInputs() throws IOException {
-        final String hg00096 = largeFileTestDir + "gvcfs/HG00096.g.vcf.gz";
-        final String hg00268 = largeFileTestDir + "gvcfs/HG00268.g.vcf.gz";
-        final String na19625 = largeFileTestDir + "gvcfs/NA19625.g.vcf.gz";
-        final String combined = largeFileTestDir + "gvcfs/combined.gatk3.7.g.vcf.gz";
-        final SimpleInterval interval = new SimpleInterval("chr20", 17960187, 17981445);
+    public void testSampleMappingFileInsteadOfVCFs() throws IOException {
+        final String sampleFileContents = "HG00096\t" + HG_00096 +"\n" +
+                "HG00268\t"+ HG_00268 + "\n" +
+                "NA19625\t"+ NA_19625;
+        final File sampleNameFile = IOUtils.writeTempFile(sampleFileContents, "sampleNameMap", ".tx");
+
+        final String workspace = createTempDir("gendbtest").getAbsolutePath() + "/workspace";
+
+        final ArgumentsBuilder args = new ArgumentsBuilder()
+                .addArgument(GenomicsDBImport.SAMPLE_NAME_MAP_LONG_NAME, sampleNameFile.getAbsolutePath())
+                .addArgument("L", IntervalUtils.locatableToString(INTERVAL))
+                .addArgument(GenomicsDBImport.WORKSPACE_ARG_NAME, workspace);
 
-        testGenomicsDBImporter(Arrays.asList(hg00096, hg00268, na19625), interval, combined);
+        runCommandLine(args);
+        checkJSONFilesAreWritten(workspace);
+        checkGenomicsDBAgainstExpected(workspace, INTERVAL, COMBINED);
+    }
+
+    @Test(expectedExceptions = CommandLineException.class)
+    public void testCantSpecifyVCFAndSampleNameFile(){
+        final ArgumentsBuilder args = new ArgumentsBuilder()
+                .addArgument(GenomicsDBImport.SAMPLE_NAME_MAP_LONG_NAME, "someFile")
+                .addArgument(StandardArgumentDefinitions.VARIANT_LONG_NAME, "someotherfile")
```

Comment:
Test would be more legit if these files actually existed (to prove that the error comes from specifying both args and not from file not found).